board.h

```diff
@@ -9,29 +9,35 @@
 #include "sphere.h"
 
 /**
- * @brief 
+ * Represents a game board containing spheres.
```

Comment:
Link relevant information, e.g.:

```suggestion
 * Represents a game board containing spheres (see `sphere.h`).
```